jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
I think this might have been one of my VS Code addons... let me explore that. I thought I had removed this from the commit.